### PR TITLE
instructions for the students application

### PR DESCRIPTION
**Pull request by kikobr82**
# Description

Exercise (e.g. C01-E01): 

Student Name:

> A brief explanation of what has been done.

## Type of change

- [ ] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +42,13 @@ To complete the DevOps Academy, you will have to go through 8 classes with hands
 
 You will also be required to peer review at least 5 pull requests from other students to finish this course.
 
+From time to time we also run the live/online classes of the Academy content with a selected group of students. Those classes uses this same repository and have no additional content. The only benefit is to have the opportunity to interact with a group of students as well as with the instructors in a more structured learning process. 
+
+Because in this live/online mode the classes occurs every fortnighly, the pace of the exercises completion are based on the course schedule, so students requires a considerable amount of hours available during the week to be able to keep the course pace and complete exercises. 
+
+Also, considering the limited number of stundets that are accepted for each live/online term, we ask people to only apply for those classes if they will be able to commit to have a 100% class attendance and are willing to invest their time in the learning process by doing the exercises and complete th group projects.
```
Comment (caiocezart) [inline README.md:49]: s/stundets/students

File: README.md [hunk 2]
```diff
@@ -55,6 +63,22 @@ Please find below the requirements to complete this course:
 
 If you need help in any of these topics, [a tutorial is available on the pre-class](classes/00pre-class/README.md):
 
+# Live/Online Applications
+
+If you intend to apply for the Live/Online classes, you'll need to fill the form available [here](https://www.devopsacademy.com.au), and as part of your application, you'll need to submit a PR to this repository under the applications folder following the instructions below. If there is no form available on the link provided, there is no scheduled classes in place. Reach out to any of us through our Slack channel to get more information on next Live/Online classes.
```
Comment (caiocezart) [inline README.md:68]: can you make some space after the first paragraph to be more clear about when no registration is open

Comment (caiocezart) [inline README.md:68]: maybe also add something like

Your application won't be accessed if a pull request with your requirements is provided. It's essential that you have a proper environment setup before being able to undergo the DevOps Academy.

File: README.md (hunk 1 above)
Comment (kikobr82) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (kikobr82) [reply README.md]: fixed

Comment (kikobr82) [reply README.md]: fixed

**Pull request merged**